test/unit/controllers/movements/ConsignmentReferencesControllerSpec.scala

```diff
@@ -54,7 +54,13 @@ class ConsignmentlReferencesControllerSpec extends ControllerLayerSpec with Mock
   private def theResponseForm: Form[ConsignmentReferences] = {
     val captor = ArgumentCaptor.forClass(classOf[Form[ConsignmentReferences]])
     verify(page).apply(captor.capture())(any(), any())
-    captor.getValue()
+    captor.getValue
+  }
+
+  private def theCache: Cache = {
```

Comment:
You dont need this `MockCache` gives you `theCacheUpserted`

Reply:
👍